install-and-configure/install/multi-cluster/federated-etl/aggregator.md

```diff
@@ -69,88 +69,69 @@ Because the Aggregator PV is relatively small, the least expensive performance g
 
 ```yaml
 kubecostAggregator:
-  # deployMethod determines how Aggregator is deployed. Current options are
-  # "singlepod" (within cost-analyzer Pod) "statefulset" (separate
-  # StatefulSet), and "disabled". Only use "disabled" if this is a secondary
-  # Federated ETL cluster which does not need to answer queries.
-  deployMethod: singlepod
-
-  # fullImageName overrides the default image construction logic. The exact
-  # image provided (registry, image, tag) will be used for aggregator.
-  # fullImageName:
-  imagePullPolicy: IfNotPresent
-
-  # For legacy configuration support, `enabled: true` overrides deployMethod
-  # and causes `deployMethod: "statefulset"`
-  enabled: false
-
-  # Replicas sets the number of Aggregator replicas. It only has an effect if
-  # `deployMethod: "statefulset"`
-  replicas: 1
-
-
-  # stagingEmptyDirSizeLimit changes how large the "staging"
-  # /var/configs/waterfowl emptyDir is. It only takes effect in StatefulSet
-  # configurations of Aggregator, other configurations are unaffected.
-  #
-  # It should be set to approximately 8x the size of the largest bingen file in
-  # object storage. For example, if your largest bingen file is a daily
-  # Allocation file with size 300MiB, this value should be set to approximately
-  # 2400Mi. In most environments, the default should suffice.
-  stagingEmptyDirSizeLimit: 2Gi
-
-  # this is the number of partitions the datastore is split into for copying
-  # the higher this number, the lower the ram usage but the longer it takes for
-  # new data to show in the kubecost UI
-  # set to 0 for max partitioning (minimum possible ram usage, but the slowest)
-  # the default of 25 is sufficient for 95%+ of users. This should only be modified
-  # after consulting with Kubecost's support team
-  numDBCopyPartitions: 1
   logLevel: info
 
-  # env: has been removed to avoid unknown issues that would be caused by
-  # customizations that were required to run aggregator in previous versions
-  # extraEnv: can be used to add new environment variables to the aggregator pod
-
-  # the below settings should only be modified with support from Kubecost staff
+  # How much data to ingest from the federated store bucket, and how much data
+  # to keep in the DB before rolling the data off.
+  # 
+  # Note: If increasing this value to backfill historical data, it will take
+  # time to gradually ingest & process those historical ETL files. Consider
+  # also increasing the resources available to the aggregator as well as the
+  # dbConcurrentIngestionCount.
+  # 
+  # default: 91
+  etlDailyStoreDurationDays: 375
```

Comment:
In most enterprise use cases they will want a year of data to be retained in the DB.